File: C/arithmetic.c
/*
Task
Read two integers from STDIN and print three lines where:

The first line contains the sum of the two numbers.
The second line contains the difference of the two numbers (first - second).
The third line contains the product of the two numbers.
*/

#include <stdio.h>

int main()
{
    int a=0,b=0;

    printf("Enter an integer a and b: ");
    scanf("%d %d",&a ,&b);

    printf("%d",a+b);
    printf("\n%d",a-b);
    printf("\n%d",a*b);
    return 0;
}

File: Java/arithmetic.java
/*
Task
Read two integers from STDIN and print three lines where:

The first line contains the sum of the two numbers.
The second line contains the difference of the two numbers (first - second).
The third line contains the product of the two numbers.
*/

import java.util.Scanner;

class arithmetic{
    public static void main(String[] args) {
        int a, b;
        Scanner scan= new Scanner(System.in);
        System.out.println("Enter an integer a: ");
        a=scan.nextInt();
        System.out.println("Enter an integer b: ");
        b=scan.nextInt();
        scan.close();
        
        System.out.println("Sum: " + (a+b));
        System.out.println("Difference: " + (a-b));
        System.out.println("Multiplication: " + (a*b));
    }
}